Code/Framework/AzToolsFramework/AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.cpp

```diff
@@ -45,14 +45,15 @@ namespace UnitTest
     // - https://lists.qt-project.org/pipermail/development/2019-July/036873.html
     void MouseMove(QWidget* widget, const QPoint& initialPositionWidget, const QPoint& mouseDelta, const Qt::MouseButton mouseButton)
     {
-        QPoint nextPosition = widget->mapToGlobal(initialPositionWidget + mouseDelta);
+        QPoint localPosition = initialPositionWidget + mouseDelta;
+        QPoint globalPosition = widget->mapToGlobal(localPosition);
```

Comment:
uber nit: Could these both be `const` and be called `nextLocalPosition` and `nextGlobalPosition` respectively.

Reply:
sure